doc/api/path.markdown

```diff
@@ -93,7 +93,22 @@ path.extname('.index')
 
 ## path.format(pathObject)
 
-Returns a path string from an object, the opposite of [`path.parse`][].
+Returns a path string from an object. This is the opposite of [`path.parse`][].
+
+If `pathObject` has all expected properties, the returned string will be a
```

Comment:
May want to expand on this just a bit to say what all the expected properties are up front.